src/nwp_consumer/internal/inputs/cmc/test_client.py

```diff
@@ -0,0 +1,101 @@
+import datetime as dt
+import pathlib
+import unittest
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ._models import CMCFileInfo
+
+from .client import Client, _parseCMCFilename
+
+testClient = Client(model="gdps")
+
+
+class TestClient(unittest.TestCase):
+    def test_mapTemp(self) -> None:
+        # Test with global file
+        testFilePath: pathlib.Path = (
+            pathlib.Path(__file__).parent / "CMC_glb_VGRD_ISBL_200_latlon.15x.15_2023080900_P027.grib2"
+        )
+        out = testClient.mapTemp(p=testFilePath)
+
+        # Check latitude and longitude are injected
+        self.assertTrue("latitude" in out.coords)
+        self.assertTrue("longitude" in out.coords)
+        # Check that the dimensions are correctly ordered and renamed
+        self.assertEqual(
```

Comment:
I'd be tempted to check these have the expected sizes as well

Reply:
Sounds good